crates/rustc_codegen_spirv/src/symbols.rs

```diff
@@ -199,23 +313,23 @@ impl Symbols {
             .map(|&(a, b)| (a, SpirvAttribute::StorageClass(b)));
         let execution_models = EXECUTION_MODELS
             .iter()
-            .map(|&(a, b)| (a, SpirvAttribute::Entry(b)));
-        let custom = std::iter::once((
-            "really_unsafe_ignore_bitcasts",
-            SpirvAttribute::ReallyUnsafeIgnoreBitcasts,
-        ));
```

Comment:
why split this out into its own thing?

Reply:
I figured it was commonly used. And this logic matches descriptor_set, binding. But it probably should be reverted to how it was, in attributes.